packages/sdk/src/CLRunner.ts

```diff
@@ -1878,7 +1965,7 @@ export class CLRunner {
             }
 
             // Use scorer step to populate pre-built data (when)
-            if (round.scorerSteps && round.scorerSteps.length > 0) {
+            if (round.scorerSteps.length > 0) {
```

Comment:
Is this safe?  Imagine the null check was added here for a reason.

Reply:
Yea I used to think things were added for a reason. Anyways similar to Thomas changes, we'll find out who was breaking the types.

Reply:
Removing this adds potential instability to the system (hard failure).   I understanding wanting to find the offending code that passes a bad object, but a console error would be better.

Reply:
ok